%Error: t/t_inst_misarray_bad.v:16: VARREF 't.foo' is not an unpacked array, but is in an unpacked array context
                                  : ... In instance t.foo
                     .foo(foo));   
                          ^~~
%Error: Exiting due to
